src/serac/numerics/functional/tuple_arithmetic.hpp

```diff
@@ -202,6 +202,30 @@ SERAC_HOST_DEVICE auto promote_to_dual_when(const T& x)
   }
 }
 
+/**
+ * @tparam dualify specify whether or not the input should be made into its dual type
+ * @tparam T the type of the values passed in
+ * @tparam n how many values were passed in
+ *
+ * @brief a function that optionally (decided at compile time) converts a list of values to their dual types
+ * @param x the values to be promoted
+ */
```

Comment:
Missing return description

Reply:
Are the `@return` entries required? I usually try to describe in words what is returned in the `@brief`s, and `doxygen` isn't warning about missing `@return` statements.